### PR TITLE
[TASK] Add/Remove flags on settings page in offline mode (DC-33)

### PR DESCRIPTION
**Pull request by mrajvanshy**

### CODE REVIEW

File: src/templates/tabs/settingsgateway.jade [hunk 1]
```diff
@@ -6,26 +6,27 @@ section.col-xs-12.content(ng-controller="SettingsGatewayCtrl")
...
       .unfunded.literal(l10n)
         | Your account has to be&#32;
         a(href="https://ripple.com/knowledge_center/activating-your-wallet/", target="_blank", l10n-inc) activated
         | &#32;before you can see this page.
 
-    .col-sm-9.list(ng-show='connected && account.Balance')
+    .col-sm-9.list(ng-show='((connected && account.Balance) || !onlineMode)')
+      //- Offline Flag Change
+      div(ng-show='offlineSettingsChange')
+        signed-transaction(data="{{signedTransaction}}", sequence="{{send.sequence}}")
```
Comment (MatthewPhinney) [inline src/templates/tabs/settingsgateway.jade]: Is `send.sequence` defined in settingsgateway.js?  Didn't see it, but I might have missed it.

**Comment by MatthewPhinney** [comment]
A lot of code is repeated within the `switch` statements. Can you pull the repeated code outside the `switch` block?

**Comment by mrajvanshy** [comment]
@MatthewPhinney not really sure what code I could pull out of the `switch` statements because it's all specific to that case, but do you have any suggestions?

**Comment by mrajvanshy** [comment]
@MatthewPhinney How does this look?

**Comment by MatthewPhinney** [comment]
LGTM

**Pull request merged**